tdrs-backend/vault/vault-integration-guide.md

```diff
@@ -0,0 +1,156 @@
+# HashiCorp Vault Integration Assessment
```

Comment:
I don't like the directory structure. Would like to put everything under one directory